Marlin/Configuration.h

```diff
@@ -2077,6 +2077,8 @@
 
 //
 // DGUS Touch Display with DWIN OS. (Choose one.)
+// ORIGIN : https://www.aliexpress.com/item/32993409517.html
+// FYSETC : https://www.aliexpress.com/item/32961471929.html
```

Comment:
I wonder if there are data sheets for these. Or are they just common DGUS / DWIN displays with custom graphics and onboard code?